sdk/core/azure-core/src/main/java/com/azure/core/http/policy/HttpLogOptions.java

```diff
@@ -202,12 +210,92 @@ public boolean isPrettyPrintBody() {
     /**
      * Sets flag to allow pretty printing of message bodies.
      *
-     * @param prettyPrintBody If true, pretty prints message bodies when logging. If the detailLevel does not
-     *                        include body logging, this flag does nothing.
+     * @param prettyPrintBody If true, pretty prints message bodies when logging. If the detailLevel does not include
+     * body logging, this flag does nothing.
      * @return The updated HttpLogOptions object.
      */
     public HttpLogOptions setPrettyPrintBody(boolean prettyPrintBody) {
         this.prettyPrintBody = prettyPrintBody;
         return this;
     }
+
+    /**
+     * Gets the {@link LogLevel} used by default when logging requests and responses.
+     * <p>
+     * {@link HttpRequestLogger#getLogLevel(LogLevel, HttpPipelineCallContext)} and {@link
+     * HttpResponseLogger#getLogLevel(LogLevel, HttpResponse, Duration)} can be used to set the {@link LogLevel} for
+     * each request and response being logged.
+     * <p>
+     * By default {@link LogLevel#INFORMATIONAL} is used.
+     *
+     * @return The {@link LogLevel} used by default when logging requests and responses.
+     */
+    public LogLevel getDefaultLogLevel() {
+        return defaultLogLevel;
+    }
+
+    /**
+     * Sets the {@link LogLevel} used by default when logging requests and responses.
+     * <p>
+     * {@link HttpRequestLogger#getLogLevel(LogLevel, HttpPipelineCallContext)} and {@link
+     * HttpResponseLogger#getLogLevel(LogLevel, HttpResponse, Duration)} can be used to set the {@link LogLevel} for
+     * each request and response being logged.
+     * <p>
+     * By default {@link LogLevel#INFORMATIONAL} is used.
+     *
+     * @param defaultLogLevel The default log level.
+     * @return The updated HttpLogOptions object.
+     */
+    public HttpLogOptions setDefaultLogLevel(LogLevel defaultLogLevel) {
+        this.defaultLogLevel = defaultLogLevel;
+        return this;
+    }
```

Comment:
This could be added in a non-breaking way in the future. Main purpose was convenience if someone wanted to have all logging done at the VERBOSE level so their log4j, etc, configurations included INFO and request/response logging was too verbose.